src/middlewares/task.middleware.js

```diff
@@ -261,9 +289,12 @@ const getDevServerCommand = (
   projectType: ProjectType,
   port: string
 ) => {
+  let command;
   switch (projectType) {
     case 'create-react-app':
-      return [`PORT=${port} npm`, 'run', task.name];
+      // For Windows Support
+      command = isWin ? `set PORT=${port} && npm` : `PORT=${port} npm`;
```

Comment:
Not urgent, but I think it'd be neat if we had a helper method, like `formatCommandForPlatform`, that looked like:

```
command = formatCommandForPlatform('npm`, { PORT: port });
```

That way, the whole `set` prefix (and even the whole prefixing with env variables) wouldn't need to be understood by devs making changes to any part of the code that starts commands.

Feel free to just add a TODO though, I can tackle this in a subsequent PR :)